server/chat-plugins/calculator.ts

```diff
@@ -132,17 +139,26 @@ export const commands: ChatCommands = {
 	math: "calculate",
 	calculate(target, room, user) {
 		if (!target) return this.parse('/help calculate');
+		const targets = target.split(',');
+		const expression = targets[0];
+		let base = parseInt(targets[1]);
+		if (isNaN(base)) base = 10;
+		if (targets.length > 1 && !(base in BASE_PREFIXES)) {
+			return this.errorReply("Invalid base. Valid options are: 2 8 10 16");
+		}
```

Comment:
Oh! You only support bases 2, 8, 10, and 16. All the more reason to use the text version, so it's clearer what bases are supported.

Reply:
I'm not sure how obvious it would be to people that they can do "in hex" or something like that. But I guess it would be a niche feature anyway.

Reply:
It can still be documented. I don't see how the comma is any more obvious?

Reply:
Because comma-separated arguments is what just about every other PS command does.

Reply:
Yes, but it's not immediately clear what the second argument _means_.

Reply:
(Like it's not immediately obvious to me that `/calc 1, 2`  means "show me 1 in base 2")